.github/workflows/multi-arch-ghcr.yaml

```diff
@@ -0,0 +1,105 @@
+name: Create and publish a Docker multi-arch image to ghcr
+
+on:
+  push:
+    tags:        
+      - v0.*
+
+env:
+  REGISTRY: ghcr.io
+  CONTROLLER_IMAGE_NAME: ${{ github.repository }}-controller
+  WATCHER_IMAGE_NAME: ${{ github.repository }}-watcher
+  WEBHOOK_IMAGE_NAME: ${{ github.repository }}-webhook
+  TKN_PAC_IMAGE_NAME: ${{ github.repository }}-tkn-pac
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@f4ef78c080cd8ba55a85445d5b36e214a81df20a
```

Comment:
can you use tags here? this allows dependabot to keep those updated?

(or perhaps this works with sha?)